CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 Nothing Yet!
 
+# Version 0.24.2 (2024-10-31)
+
+This release contains a few new features.
+
+## Improvements to installer customization
+
+We now provide two new environment variables making it possible to customize just the base URL for fetching installers. Previously, the only customization for overriding the install URL overrode the full URL, including version-specific paths, which made it difficult to use for users who were proxying or mirroring GitHub paths. These new variables are branded with your app's name. For example:
```

Comment:
TODO: link to docs

Reply:
we should also make sure to call out the contributor on this even tho his PR was to the updater

Reply:
Very good call.

Reply:
https://opensource.axo.dev/cargo-dist/book/installers/usage.html#artifact-location